content/realtime/channels/overview.textile

```diff
@@ -155,8 +155,13 @@ Channel parameters allow a client to set the properties of a channel. The are tw
 * "delta":/realtime/channels/channel-parameters/deltas
 
 bc[jsall]. const realtime = new Ably.Realtime('{{API_KEY}}');
-const channelOpts = {params: {rewind: '1'}}
-const channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+const channelOptions = {
+  params: {
+    delta: 'vcdiff',
+    rewind: '1'
+  }
+};
+const channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOptions);
```

Comment:
@QuintinWillison can I just verify that this is correct?

Reply:
See https://github.com/ably/docs/pull/880#issuecomment-637452883